Objects/object.c

```diff
@@ -2279,6 +2279,33 @@ Py_XNewRef(PyObject *obj)
     return _Py_XNewRef(obj);
 }
 
+#undef Py_Is
+#undef Py_IsNone
+#undef Py_IsTrue
+#undef Py_IsFalse
+
+// Export Py_Is(), Py_IsNone(), Py_IsTrue(), Py_IsFalse() as regular functions
```

Comment:
I am not super conviced. Why would people would like to use `Py_Is` if is going to be slower than just pointer comparison?

Reply:
The intent is correctness if you care about other Python implelentations than CPython. If you don't care, well, don't use it :-)